### DISCUSSION TITLE
zfs-windows-2.2.6-rc9

### DISCUSSION THREAD

**Discussion by lundman** [openzfsonwindows/openzfs#413] [Development]
Bring in the verdict on rc9, let me know what things are still broken. You don't want me with nothing to do for long.

I as curious about the long filename patch, but if I sync up, we will be 3.0rc from now, and the feedback was to hold back on too-new features.

**Comment by guenther-alka** [comment]
List several snaps works now but snaps still empty when should not

```
PS C:\Users\me> dir l:\data


    Verzeichnis: l:\data


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        01.11.2024     09:58              0 neu.txt


PS C:\Users\me> zfs snap tank/data@4
PS C:\Users\me> del l:\data\neu.txt
PS C:\Users\me> dir l:\data
PS C:\Users\me> dir l:\data\.zfs\snapshot\4
PS C:\Users\me>
```

Netcat replication from/to Windows/Proxmox via nc64 is also ok (had problems with rc8)
https://github.com/openzfsonwindows/openzfs/discussions/408#discussioncomment-11059259

Do not forget to use cmd as pipes have problems in Powershell

**Comment by lundman** [comment]
```
PS C:\Users\me> zfs snap tank/data@4
PS C:\Users\me> del l:\data\neu.txt
PS C:\Users\me> dir l:\data

zfs mount tank/data@4

PS C:\Users\me> dir l:\data\.zfs\snapshot\4
PS C:\Users\me>
```

**Reply by guenther-alka** [reply]
ok, the extra zfs mount tank/data@4 solves the problem
(I am not used to this step as on Illumos/Solaris the snap folder is simply always up to data)

Is there a way, a snap can just work/show current state when opening in .zfs/snapshot?

**Reply by lundman** [reply]
Basically while the mount/unmount code was iffy, I disabled the automounting of snapshots, since you can too easily trigger a mass mount.
But we can go back and add automounting again in the Windows port.

**Comment by guenther-alka** [comment]
Am I correct when I say
OpenZFS on Windows 2.2.6 rc9 is the first "quite usable" release with only minor problems left?

As it includes newer commits from upstream OpenZFS Master than are in the stable OpenZFS latest, I wait for an integration of OpenZFS 2.3 when it becomes the stable latest.

**Comment by guenther-alka** [comment]
I have seen a report about crashes with local Crystal Disk benchmark on ZFS
https://www.reddit.com/r/zfs/comments/1gf9jef/comment/lxhasxy/?context=3

**Comment by besterino** [comment]
Oi, just posted a similar prob here: https://github.com/openzfsonwindows/openzfs/discussions/416
